src/main/java/com/juick/android/widget/util/ImageUtil.kt

```
@@ -15,19 +15,17 @@
 
-class DiscussionsFragment : FeedFragment() {
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-        if (vm.apiUrl.value == Uri.EMPTY) {
-            vm.apiUrl.value = Uris.discussions
-        }
+suspend fun loadImage(url: String): Bitmap? {
+    return try {
+        val responseBody = App.instance.api.download(url)
+        responseBody.byteStream().use { BitmapFactory.decodeStream(it) }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)